src/System.CommandLine.Tests/VersionOptionTests.cs

```diff
@@ -156,15 +158,17 @@ public void Version_option_is_not_added_to_subcommands()
         [Fact]
         public async Task Version_not_added_if_it_exists()
         {
-            // Adding an option multiple times can occur two ways in
-            // real world scenarios - invocation can be invoked twice
-            // or the author may have their own version switch but
-            // still want other defaults.
-            var configuration = new CommandLineBuilder(new RootCommand())
-                         .UseVersionOption()
-                         .UseVersionOption()
-                         .Build();
-            configuration.Output = new StringWriter();
+            // The RootCommand ctors adds VersionOption by default.
+            // Users might not know if or just want to customize it.
+            RootCommand commandWithDuplicates = new()
+            {
+                new VersionOption(),
+                new VersionOption(),
```

Comment:
Shouldn't this be an error because of the symbol identifier collision?

I think the expectation with the builder is that it's idempotent, but I don't think I have the same expectation with `Command.Add` since we're enforcing name and alias uniqueness.